### ISSUE TITLE
Training failed

### ISSUE THREAD

**Issue by whoyouwith91** [connorcoley/conv_qsar_fast#2]
Hi, when I run the script "python conv_qsar_fast/main/main_cv.py conv_qsar_fast/inputs/De-aq.cfg", some minor errors show up. I fixed it and then it runs normally. However, the final results are odd. The training loss never decreases in each CV folder and the predicted values stay all the same. I run it several times but with same results. Don't know why.

**Comment by connorcoley** [comment]
Hi,

What were those errors and how did you fix them? It's hard to know what the problem could be without any other information.  I imagine that you are using a newer version of Keras than this code was written for and that could be the root cause.
